### ISSUE TITLE
platform-api incompatible with heroics 0.0.22

### ISSUE THREAD

**Issue by davidcpell** [heroku/platform-api#71]
I was just going through an app to update dependencies and think I've found an incompatibility. `platform-api` depends on `~> 0.0.21` of `heroics`, which just today released `0.0.22`. Running `bundle update` in the project updates the `heroics` dependency to `0.0.22` and keeps `bundler` from being able to load gems:

```
# Gemfile.lock
    heroics (0.0.22)
      erubis (~> 2.0)
      excon
      multi_json (>= 1.9.2)

    ...
    platform-api (1.0.1)
      heroics (~> 0.0.21)
      moneta (~> 0.8.1)
```

```
➜  app git:(gem-update-2017-05) ✗ bundle exec rake
rake aborted!
Bundler::GemRequireError: There was an error while trying to load the gem 'platform-api'.
Gem Load Error is: Duplicate link names 'info', 'update', 'delete'.
Backtrace for gem load error is:
/Users/davidpell/.gem/ruby/2.3.1/gems/heroics-0.0.22/lib/heroics/schema.rb:68:in `initialize'
/Users/davidpell/.gem/ruby/2.3.1/gems/heroics-0.0.22/lib/heroics/schema.rb:14:in `new'
/Users/davidpell/.gem/ruby/2.3.1/gems/heroics-0.0.22/lib/heroics/schema.rb:14:in `block in initialize'
/Users/davidpell/.gem/ruby/2.3.1/gems/heroics-0.0.22/lib/heroics/schema.rb:13:in `each'
/Users/davidpell/.gem/ruby/2.3.1/gems/heroics-0.0.22/lib/heroics/schema.rb:13:in `initialize'
/Users/davidpell/.gem/ruby/2.3.1/gems/heroics-0.0.22/lib/heroics/configuration.rb:36:in `new'
/Users/davidpell/.gem/ruby/2.3.1/gems/heroics-0.0.22/lib/heroics/configuration.rb:36:in `schema_filepath='
/Users/davidpell/.gem/ruby/2.3.1/gems/platform-api-1.0.1/config/client-config.rb:7:in `block in <top (required)>'
/Users/davidpell/.gem/ruby/2.3.1/gems/heroics-0.0.22/lib/heroics.rb:15:in `tap'
/Users/davidpell/.gem/ruby/2.3.1/gems/heroics-0.0.22/lib/heroics.rb:15:in `default_configuration'
/Users/davidpell/.gem/ruby/2.3.1/gems/platform-api-1.0.1/config/client-config.rb:4:in `<top (required)>'
/Users/davidpell/.gem/ruby/2.3.1/gems/platform-api-1.0.1/lib/platform-api.rb:4:in `require_relative'
/Users/davidpell/.gem/ruby/2.3.1/gems/platform-api-1.0.1/lib/platform-api.rb:4:in `<top (required)>'
/Users/davidpell/.gem/ruby/2.3.1/gems/bundler-1.14.6/lib/bundler/runtime.rb:91:in `require'
/Users/davidpell/.gem/ruby/2.3.1/gems/bundler-1.14.6/lib/bundler/runtime.rb:91:in `block (2 levels) in require'
/Users/davidpell/.gem/ruby/2.3.1/gems/bundler-1.14.6/lib/bundler/runtime.rb:86:in `each'
/Users/davidpell/.gem/ruby/2.3.1/gems/bundler-1.14.6/lib/bundler/runtime.rb:86:in `block in require'
/Users/davidpell/.gem/ruby/2.3.1/gems/bundler-1.14.6/lib/bundler/runtime.rb:75:in `each'
/Users/davidpell/.gem/ruby/2.3.1/gems/bundler-1.14.6/lib/bundler/runtime.rb:75:in `require'
/Users/davidpell/.gem/ruby/2.3.1/gems/bundler-1.14.6/lib/bundler.rb:107:in `require'
```

The error does not occur if I force `gem 'heroics', '= 0.0.21'` in `Gemfile`.

**Comment by gudmundur** [comment]
Nice to see this actually fail, given that Heroics 0.0.22 was just released a couple of hours ago to raise that error.

**Comment by geemus** [comment]
I believe this should be fixed by #70, which I hope to get merged shortly.
